scripts/ci-k8s-e2e-node-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+PROW_BUILD_ID="${BUILD_ID:-000000000000}"
+AZURE_RESOURCE_GROUP="win-e2e-node-${PROW_BUILD_ID}"
+AZURE_DEFAULT_IMG="MicrosoftWindowsServer:WindowsServer:2022-datacenter-core-smalldisk-g2:latest"
+
+SCRIPT_PATH=$(realpath "${BASH_SOURCE[0]}")
+SCRIPT_ROOT=$(dirname "${SCRIPT_PATH}")
+source ${SCRIPT_ROOT}/ci-k8s-common.sh
+
+ensure_azure_cli
+build_resource_group
+build_test_vm
+generate_ssh_key
+enable_ssh_windows
+test_ssh_connection
+echo "Test VM created. SSH connection working"
+
+echo "Install container features in VM"
+run_remote_cmd ${VM_PUB_IP} ${SSH_KEY_FILE} "powershell.exe -command { Install-WindowsFeature -Name 'Containers' -Restart }"
+wait_for_vm_restart
+
+copy_to ./scripts/prepare_env_windows.ps1 '/prepare_env_windows.ps1' ${VM_PUB_IP}
+copy_to ./scripts/prepare_e2e_node_windows.ps1 '/prepare_e2e_node_windows.ps1' ${VM_PUB_IP}
+copy_to ./scripts/k8s_e2e_node_windows.ps1 '/k8s_e2e_node_windows.ps1' ${VM_PUB_IP}
+
+run_remote_cmd ${VM_PUB_IP} ${SSH_KEY_FILE} 'c:/prepare_env_windows.ps1'
+run_remote_cmd ${VM_PUB_IP} ${SSH_KEY_FILE} 'c:/prepare_e2e_node_windows.ps1'
+
+if [ "${JOB_TYPE}" == "presubmit" ]
+then
+    echo "Running a presubmit job"
+    run_remote_cmd ${VM_PUB_IP} ${SSH_KEY_FILE} "c:/k8s_e2e_node_windows.ps1 -repoName ${REPO_NAME} -repoOrg ${REPO_OWNER} -pullRequestNo ${PULL_NUMBER} -pullBaseRef ${PULL_BASE_REF} ${test_packages_arg}"
+    exit_code=$?
+else
+    echo "Running periodic job"
+    run_remote_cmd ${VM_PUB_IP} ${SSH_KEY_FILE} 'c:/k8s_e2e_node_windows.ps1'
+    exit_code=$?
+fi
+copy_from 'c:/Logs/*.xml' ${ARTIFACTS} ${VM_PUB_IP}
```

Comment:
Is it safe to hard code this directory? Should this be an env var?

Reply:
It looks like the directory is hardcoded in `k8s_e2e_node_windows.ps1` so this should be *OK*
Might be better to pass the log directory to `k8s_e2e_node_windows.ps1` from this script so there won't ever be drift but maybe we can just add a TODO here and address it if it causes any options